### ISSUE TITLE
Capability Questions

### ISSUE THREAD

**Issue by glsutter** [chaosblade-io/chaosblade#449]
## Issue Description

Question

### Describe what happened (or what feature you want)

Trying to evaluate ChaosBlade as an option for resiliency testing. But I'm not sure if this is a feature request or a question. Actually, two questions:
- Does ChaosBlade support Azure, or can it be extended to support Azure?
- Can ChaosBlade inject failures into a Platform as a Service (PaaS) component? For example, change the memory quota associated with an Azure Function? That would seem to require use of one of the Azure APIs or maybe PowerShell. 

### How to reproduce it (as minimally and precisely as possible)

1. 
2. 
3. 

### Tell us your environment

Azure, mostly PaaS components like Azure Functions and Cosmos DB.

**Comment by xcaspar** [comment]
@glsutter Yes, we plan to develop a new project that specifically invokes the cloud service api to achieve cloud service fault injection. We will open source soon. 👍

**Comment by oss-sentinel-ai** [comment]
**Issue Evaluation**

Category: `type/feature` | Status: **Evaluated** | `good first issue`

Questions about ChaosBlade's capability:
1. Does ChaosBlade support Azure?
2. Can ChaosBlade inject failures into PaaS services?

**Status (from @xcaspar):** A new project for cloud service API-based fault injection was planned. This has been partially addressed by:
- **ChaosBlade-Box:** Provides a platform for managing experiments across multiple environments
- **ChaosBlade-Cloud:** Experimental cloud provider integration (AWS ECS/EBS)

**Azure Support:**
- ChaosBlade can run on Azure VMs for OS-level experiments
- Azure Kubernetes Service (AKS) is supported via the K8s operator
- Azure-specific PaaS (e.g., Azure Functions, Cosmos DB) is not directly supported

**PaaS Fault Injection:**
- Use ChaosBlade Box with custom plugins for PaaS services
- Or use the `script` executor to call Azure CLI/SDK:
  ```bash
  blade create script delay --time 3000 --script-path /path/to/azure-fault.sh
  ```

**Related:** #828 (custom experiments), #499 (observability integration)

---
*Automated evaluation by [oss-taishan-ai](https://github.com/oss-taishan-ai) - ChaosBlade Community Bot*
